packages/flex-plugin-scripts/src/scripts/check-start.ts

```diff
@@ -181,6 +189,42 @@ export const _checkPluginCount = () => {
   }
 };
 
+/**
+ * Touch ~/.twilio-cli/flex/plugins.json if it does not exist
+ * Check if this plugin is in this config file. If not, add it.
+ * @private
+ */
+export const _checkPluginConfigurationExists = async() => {
+  // Check to see if the directory exists & if not, create it
```

Comment:
Can you remove some of these comments? They are too verbose - the code is very clean is self explanatory.